share/smack/top.py

```diff
@@ -110,6 +110,9 @@ def arguments():
   noise_group.add_argument('--debug-only', metavar='MODULES', default=None,
     type=str, help='limit debugging output to given MODULES')
 
+  noise_group.add_argument('--warn', metavar='OPTION', default="unsound",
```

Comment:
What are the available options? You should restrict them, which is possible in python command line parser.